crypto/src/main/kotlin/net/corda/v5/crypto/DigitalSignature.kt

```diff
@@ -36,4 +36,14 @@ open class DigitalSignature(
          */
         val context: Map<String, String>
     ) : DigitalSignature(bytes)
+
+    class WithId(
+        /**
+         * SHA-256 digest of the public key. TODO add reasoning for hashing public hey
+         */
+        val by: SecureHash,
+        bytes: ByteArray
+        // TODO do we need to include the context map?
```

Comment:
Suggest we just delete this and the commented out context map for now.

Reply:
Done